[Interactive] Integrate Recommendation capability into `interactive` (#5727)
Azure/azure-cli-extensions: src/interactive/HISTORY.rst

Release History
===============

0.4.7
+++++

> * Add a recommender that predicts the next command a user might execute.

Reviewer: ```suggestion
* Support command recommendations that predicts the next command a user might need.
* Support scenario recommendations to help users complete the complex scenarios more smoothly and simply.
```

Release History
===============

0.4.7
+++++
* Support command recommendations that predicts the next commands users might need.

> * Support scenario recommendations to help users complete the complex scenarios more smoothly and simply.

Reviewer: ```suggestion
* Support scenario recommendations to help users complete the complex scenarios more smoothly and simply.
* Support new config `az config set interactive.enable_recommender=False` to disable these new recommenders.
```

> * Support command recommendations that predicts the next commands users might need.
> * Support scenario recommendations to help users complete the complex scenarios more smoothly and simply.
> * Fix display offset in toolbar.

Reviewer: ```suggestion
* Support command recommendations that predicts the next commands users might need.
* Support scenario recommendations to help users complete the complex scenarios more smoothly and simply.
* Support new config `az config set interactive.enable_recommender=False` to support disabling these new recommenders.
* Fix display offset in toolbar.
```
